biochemist-python/chapters/introduction.ipynb

```diff
@@ -85,23 +85,12 @@
   },
```

Comment:
<p>I'm not a fan of the variable name <code>v_nmolspermin</code>.  I think it is still long and less clear. What do you think of using something like <code>v_per_min</code>  and putting a comment the units. </p><p> </p><pre class="ql-syntax" spellcheck="false">velocity = velocity / 60  # units = nmol/sec
v_per_min = velocity * 60 # units = nmol/min
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>